src/pages/workspaces/workspace/ShareWorkspaceModal.js

```diff
@@ -121,7 +121,8 @@ const ShareWorkspaceModal = ({ onDismiss, workspace, workspace: { workspace: { n
           value: aclItem,
           onChange: v => setAcl(_.set([index], v)),
           disabled,
-          maxAccessLevel: workspace.accessLevel
+          maxAccessLevel: workspace.accessLevel,
+          isAzureWorkspace: !!workspace?.azureContext
```

Comment:
As noted above, this is suspect because `isAzureWorkspace` will be false if workspace is undefined. So either you have to make sure that `workspace` is not undefined at this point (in which case you shouldn't be using `?`), or you have to handle that case so that potentially problematic elements aren't displayed.